### PR TITLE
[connector/signaltometrics] Add readme and update metadata with git repo

### PR DESCRIPTION
**Pull request by lahsivjar**

### CODE REVIEW

File: connector/signaltometricsconnector/README.md [hunk 1]
```diff
@@ -1,3 +1,212 @@
...
+
+[Exporter Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#exporter-pipeline-type
+[Receiver Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#receiver-pipeline-type
+[Stability Level]: https://github.com/open-telemetry/opentelemetry-collector#stability-levels
+<!-- end autogenerated section -->
+
+## Configuration
+
+The component requires a configuration section to work. The component can produce
+metrics from spans, datapoints (for metrics), and logs. Atleast one of the metrics
```
Comment (carsonip) [inline connector/signaltometricsconnector/README.md:29]:
```suggestion
metrics from spans, datapoints (for metrics), and logs. At least one of the metrics
```

File: connector/signaltometricsconnector/README.md [hunk 2]
```diff
@@ -1,3 +1,212 @@
...
+- `resource.bar` will always be present because of the `default_value`. If the incoming
+  data does not have a resource attribute with name `resource.bar` then the configured
+  `default_value` of `bar` will be used.
+
+### Single writer
+
+Metrics data streams MUST obey [single-writer](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#single-writer) principal. However,
+since `signaltometrics` component produces metrics from all signal types and also
+allows customizing the resource attributes, there is a possibility of violating the
+single-writer principal. To keep the single-writer principal entact, the component adds
```
Comment (carsonip) [inline connector/signaltometricsconnector/README.md:194]:
```suggestion
single-writer principal. To keep the single-writer principle entact, the component adds
```

**Review by carsonip — APPROVED** [review]
nit

File: connector/signaltometricsconnector/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lahsivjar) [reply connector/signaltometricsconnector/README.md]: Lol principal 🤦 .

> enact? intact?
I meant `intact`, fixed.

**Review by carsonip — APPROVED** [review]

**Pull request merged**